### DISCUSSION TITLE
Debugging of code built on other machines

### DISCUSSION THREAD

**Discussion by jac-usai** [touchlab/xcode-kotlin#92] [Q&A]
Hi @kpgalligan , just wanted to know if there is an update about the last message you added here:
https://github.com/touchlab/xcode-kotlin/issues/31

I'm looking for a way to debug code compiled in our CI mainly, and also I'm interested in a more specific use case:
I have a multiplatform container module that embeds some other multiplatform modules from maven. The container module doesn't have any code inside, it just exports its dependencies, as described [here](https://kotlinlang.org/docs/multiplatform-build-native-binaries.html#export-dependencies-to-binaries).

This means that even if I build the container module locally, it's impossible for the debugger to see the code, since it isn't actually part of the container module project.

**Comment by kpgalligan** [comment]
We've figured this out as proof-of-concept, but haven't finished anything to do it. You need to configure the CI builds to have a know base path, then remap locally. We struggled to find a good use case, but yours may be.

**Comment by jac-usai** [comment]
@kpgalligan Do you think I can help you somehow?
